clients/cpp-console/include/AudioPlayerStream.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include <cstddef>
+
+#pragma once
+
+class IAudioPlayerStream
+{
+public:
+    enum class AudioPlayerStreamType
+    {
+        PULLAUDIOOUTPUTSTREAM,
```

Comment:
I would put underscores between the words here

Reply:
Done